guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p>The <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=from_feather#arcgis.features.GeoAccessor.to_featureclass" target="_blank">to_featureclass()</a></code> method can be used to export spatial data from a <em>SeDF</em> into various local databases, such as a <a href="https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/manage-file-gdb/file-geodatabases.htm" target="_blank">File geodatabase</a>, a <a href="https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/manage-mobile-gdb/mobile-geodatabases.htm" target="_blank">Mobile geodatabase</a> (.geodatabase), or a <a href="https://pro.arcgis.com/en/pro-app/latest/tool-reference/data-management/create-sqlite-database.htm" target="_blank">SQLite Database</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>